go-controller/pkg/ovn/policy.go

```diff
@@ -1492,3 +1495,80 @@ func (oc *Controller) shutdownHandlers(np *networkPolicy) {
 		oc.watchFactory.RemoveServiceHandler(handler)
 	}
 }
+
+// iterateRetryNetworkPolicies checks if any outstanding NetworkPolicies exist
+// then tries to re-add them if so
+// updateAll forces all policies to be attempted to be retried regardless
+func (oc *Controller) iterateRetryNetworkPolicies(updateAll bool) {
+	oc.retryNetPolices.retryMutex.Lock()
+	defer oc.retryNetPolices.retryMutex.Unlock()
+	now := time.Now()
+	for namespacedName, npEntry := range oc.retryNetPolices.entries {
+		if npEntry.ignore {
+			continue
+		}
+		// check if we need to create
+		var policyToCreate *knet.NetworkPolicy
```

Comment:
same as I mentioned for nodesretry, do we need a new variable? can't we reuse the newObj itself?